ldclient.go

```diff
@@ -66,8 +66,13 @@ var DefaultConfig = Config{
 	Offline:       false,
 }
 
-var ErrInitializationTimeout = errors.New("Timeout encountered waiting for LaunchDarkly client initialization")
-var ErrClientNotInitialized = errors.New("Toggle called before LaunchDarkly client initialization completed")
+var (
```

Comment:
Refactored these out so that clients can check against them more easily, and we can re-use them in code.